src/pages/Product/Product.js

```diff
@@ -1,28 +1,33 @@
-import React from "react";
+import React, { useState } from "react";
+import Color from "./components/Color/Color";
+import ColorButton from "./components/ColorButton/ColorButton";
+import Review from "./components/Review/Review";
+import Count from "./components/Count/Count";
 import "./Product.scss";
 
 const Product = () => {
+  const [color, setColor] = useState("white");
+  const [isReviewOpen, setIsReviewOpen] = useState(false);
+  const [number, setNumber] = useState(1);
+
   const price = 300;
-  const totalPrice = price;
+  const totalPrice = price * `${number}`;
```

Comment:
```suggestion
  const totalPrice = price * number;
```
꼭 스트링으로 변환하지 않아도 계산이 가능합니다!